### DISCUSSION TITLE
Log-likelihood test to find out which model is better

### DISCUSSION THREAD

**Discussion by MuradOmarov** [CamDavidsonPilon/lifelines#1450] [Q&A]
Hello!
I'm wondering what I should do to see if adding a single predictor improves the performance of the model significantly? I want to use not only C-index comparison. 
Do I understand correctly that if I want to compare log-likelihood of two model I should use `cph.score(test,scoring_method='log_likelihood')` for each model (trained and evaluated on the same data) then perform log-likelihood ratio test? Or should I use another objects? For example, partial log-likelihood from summary?

Thanks in advance!

**Comment by CamDavidsonPilon** [comment]
Since you are adding predicators, you're log-likelihood will increase always, so you want to use a method that takes into account the number of predictors, like the [BIC](https://en.wikipedia.org/wiki/Bayesian_information_criterion) or [AIC](https://en.wikipedia.org/wiki/Akaike_information_criterion).

**Reply by MuradOmarov** [reply]
Thanks! I agree it's a little weird. But several sources advise using log partial likelihood-based statistics for nested models. For example, https://rdrr.io/cran/survival/man/anova.coxph.html and https://www.graphpad.com/guides/prism/latest/statistics/stat_cox_regression_compare_tab.htm. So I would be very glad to hear your opinion, what do you think about anova, LRT?
